### DISCUSSION TITLE
typing/editing issue: typing is so slow when editing a better notes with substantial contents

### DISCUSSION THREAD

**Discussion by doczhengexplorealone** [windingwind/zotero-better-notes#867] [Q&A]
- OS: Windows10 22H2
- Zotero Version: 7.0.0-beta.54+6b996d4f9 (64-bit）
- Plugin Version:1.1.4-49

typing is soooo slow when editing a better notes with substantial contents(few seconds for each word input), while normal when editing a newly created notes or in other typing condition like MS word or web page. actually，zotero 6 and the corresponding version of better notes has this isssue too.

**Comment by windingwind** [comment]
very much possibly it's not an issue with BN. try:
1. disable `Check spelling` via right-click menu.
2. disable all other plugins and try again.
